terra-workspace-manager-performance/src/gatling/simulations/CreateWorkspaceSimulation.scala

```diff
@@ -0,0 +1,99 @@
+import com.google.auth.oauth2.GoogleCredentials
```

Comment:
We may benefit from moving some of the test-agnostic 'config' stuff (auth, GHA, etc) out of this test file. I understand that this is a POC, but I think it's worth mentioning.

Reply:
Yup definitely that's the plan. Integration tests also use the same library, maybe we should have a shared library that both performance and integration tests can use.